src/main/java/org/apache/geronimo/microprofile/impl/jwtauth/cdi/GeronimoJwtAuthExtension.java

```diff
@@ -85,9 +85,8 @@ void setClaimMethodsBinding(@Observes final BeforeBeanDiscovery beforeBeanDiscov
 
     void captureInjections(@Observes final ProcessInjectionPoint<?, ?> processInjectionPoint) {
         final InjectionPoint injectionPoint = processInjectionPoint.getInjectionPoint();
-        ofNullable(injectionPoint.getAnnotated().getAnnotation(Claim.class))
-                .flatMap(claim -> createInjection(claim, injectionPoint.getType()))
-                .ifPresent(injectionPoints::add);
+        ofNullable(injectionPoint.getAnnotated().getAnnotation(Claim.class)).ifPresent(
```

Comment:
Formatting issue?

Reply:
No this is a legit NPE issue.

Reply:
```
org.apache.webbeans.exception.WebBeansConfigurationException: Error while sending SystemEvent to a CDI Extension! org.apache.webbeans.portable.events.generics.GProcessInjectionPoint@53c028de
	at org.apache.webbeans.event.NotificationManager.fireEvent(NotificationManager.java:736)
	at org.apache.webbeans.container.BeanManagerImpl.fireEvent(BeanManagerImpl.java:495)
	at org.apache.webbeans.container.BeanManagerImpl.fireEvent(BeanManagerImpl.java:469)
	at org.apache.webbeans.util.WebBeansUtil.fireProcessInjectionPointEvent(WebBeansUtil.java:789)
	at org.apache.webbeans.inject.impl.InjectionPointFactory.buildInjectionPoint(InjectionPointFactory.java:183)
	at org.apache.webbeans.event.ObserverMethodImpl.<init>(ObserverMethodImpl.java:197)
	at org.apache.webbeans.event.ObserverMethodImpl.<init>(ObserverMethodImpl.java:137)
	at org.apache.webbeans.event.NotificationManager.getObservableMethodForAnnotatedMethod(NotificationManager.java:842)
	at org.apache.webbeans.component.creation.ObserverMethodsBuilder.defineObserverMethod(ObserverMethodsBuilder.java:129)
	at org.apache.webbeans.component.creation.ObserverMethodsBuilder.defineObserverMethods(ObserverMethodsBuilder.java:77)
	at org.apache.webbeans.util.WebBeansUtil.createExtensionComponent(WebBeansUtil.java:493)
	at org.apache.webbeans.portable.events.ExtensionLoader.addExtension(ExtensionLoader.java:206)
	at org.apache.aries.cdi.owb.OWBCDIContainerInitializer$1$1.lambda$loadExtensionServices$0(OWBCDIContainerInitializer.java:159)
	at java.util.IdentityHashMap.forEach(IdentityHashMap.java:1354)
	at org.apache.aries.cdi.owb.OWBCDIContainerInitializer$1$1.loadExtensionServices(OWBCDIContainerInitializer.java:159)
	at org.apache.webbeans.config.BeansDeployer.deploy(BeansDeployer.java:231)
	at org.apache.webbeans.lifecycle.AbstractLifeCycle.bootstrapApplication(AbstractLifeCycle.java:137)
	at org.apache.webbeans.lifecycle.AbstractLifeCycle.startApplication(AbstractLifeCycle.java:103)
	at org.apache.webbeans.web.lifecycle.WebContainerLifecycle.startApplication(WebContainerLifecycle.java:98)
	at org.apache.aries.cdi.owb.OWBCDIContainerInitializer.initialize(OWBCDIContainerInitializer.java:172)
	at org.apache.aries.cdi.owb.OWBCDIContainerInitializer.initialize(OWBCDIContainerInitializer.java:45)
	at org.apache.aries.cdi.container.internal.container.ContainerBootstrap.open(ContainerBootstrap.java:141)
	at org.osgi.util.promise.DeferredPromiseImpl$Submit.run(DeferredPromiseImpl.java:658)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.webbeans.exception.WebBeansException: java.lang.NullPointerException
	at org.apache.webbeans.event.ObserverMethodImpl.notify(ObserverMethodImpl.java:371)
	at org.apache.webbeans.event.NotificationManager.invokeObserverMethod(NotificationManager.java:819)
	at org.apache.webbeans.event.NotificationManager.fireEvent(NotificationManager.java:715)
	... 25 common frames omitted
Caused by: java.lang.NullPointerException: null
	at org.apache.geronimo.microprofile.impl.jwtauth.cdi.GeronimoJwtAuthExtension.captureInjections(GeronimoJwtAuthExtension.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.webbeans.event.ObserverMethodImpl.invoke(ObserverMethodImpl.java:404)
	at org.apache.webbeans.event.ContainerEventObserverMethodImpl.invoke(ContainerEventObserverMethodImpl.java:85)
	at org.apache.webbeans.event.ObserverMethodImpl.notify(ObserverMethodImpl.java:365)
	... 27 common frames omitted
```

Reply:
can you precise what is null there? flatMap maps an optional to an optional and protects against null normally so sounds like it could only be injectionpoint or ip.getType() and both shouldnt be null